docs/detections/api/lists/api-delete-list-item.asciidoc

```diff
@@ -9,6 +9,24 @@ Deletes list items.
 
 ===== URL query parameters
 
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`id` |String |Unique identifier of the list item. |Required if `list_id` and `value` are not specified.
+|`list_id` |String |ID of the <<lists-api-create-container, list container>>.
```

Comment:
Might be more consistent to say "Unique identifier" instead of "ID", like on the previous line